### ISSUE TITLE
Broken Fog?

### ISSUE THREAD

**Issue by Demez** [Demez/Quiver#36]
Occurs in the area where you get the rocket launcher on d2_coast_03

![2019-05-26_11-14-52_quiver_HALF-LIFE_2](https://user-images.githubusercontent.com/30940677/58383695-c0a7d000-7fa7-11e9-8dfa-ba4aab2c9e32.png)

**Comment by Demez** [comment]
It might have to do with this error in console:

```
unrecognized conditional test !lowfill?$startfadesize in particle/smoke1/smoke1
unrecognized conditional test lowfill?$startfadesize in particle/smoke1/smoke1
unrecognized conditional test !lowfill?$endfadesize in particle/smoke1/smoke1
unrecognized conditional test lowfill?$endfadesize in particle/smoke1/smoke1
unrecognized conditional test lowfill?$blendframes in particle/smoke1/smoke1
unrecognized conditional test !lowfill?$depthblend in particle/smoke1/smoke1
unrecognized conditional test lowfill?$depthblend in particle/smoke1/smoke1
```

**Comment by Demez** [comment]
fixed in commit 8f91f86c19f7d4e9472dac7b4e6e0017ab2f4210
